packages/server/__tests__/inviteToTeam.test.ts

```diff
@@ -19,7 +18,7 @@ test('Invite to team, works for ordinary email domain', async () => {
     `,
     variables: {
       teamId,
-      invitees: ['foo@example.com', inviteeEmail]
+      invitees: ['nick@parabol.co']
```

Comment:
We need to make sure the email addresses are valid now. The `signUp` function returns invalid emails.